pom.xml

```diff
@@ -156,6 +156,84 @@
       <artifactId>commons-logging</artifactId>
       <scope>provided</scope>
     </dependency>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This change brakes the file... you paste in the middle of this xml file another xml file.